packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -111,6 +111,37 @@ const TEMPLATE_SOURCE_NPM = 'npm'
 const TEMPLATE_SOURCE_BUNDLE = 'bundle'
 const DEFAULT_TEMPLATE_VERSION = 'latest'
 
+const askApplicationExtensibiltyQuestions = (appExtensions) => {
+    return [
+        {
+            name: 'project.generationType',
+            message:
+                'Do you want to generate a project using Application Extensibility or using the template-retail-react-app?',
```

Comment:
We can probably simplify this and just ask do you want to use application extensibility because if we talk about a particular template it implies that there are other templates that they can use which they currently can't and can only access them with presets so unless we want to add some more functionality to be able to select a different template. I think we just say hey do you want to use extensibility